modules/upgrade/pages/k-rolling-upgrade.adoc

```diff
@@ -114,21 +115,11 @@ CAUTION: Make sure to include all existing overrides, otherwise the upgrade may
 [[upgrade]]
 == Perform a rolling upgrade
 
-A rolling upgrade involves running the following procedure on each broker, one at a time:
-
-. Upgrade the version of Redpanda on the broker that's running in the Pod with the highest ordinal.
-. Place the broker into maintenance mode.
-. Wait for maintenance mode to finish.
-. Terminate the Pod so that it is recreated with the new version of Redpanda.
-. Take the broker out of maintenance mode.
-
-Placing brokers into maintenance mode ensures a smooth upgrade of your cluster while reducing the risk of interruption or degradation in service. When a broker is placed into maintenance mode, it reassigns its partition leadership to other brokers for all topics that have a replication factor greater than one (three is the default replication factor for topics). Reassigning partition leadership involves _draining_ leadership from the broker and _transferring_ that leadership to another broker.
+A rolling upgrade involves updating the version of Redpanda on each broker and performing a rolling restart.
```

Comment:
Just to confirm I'm reading this right -- you update each Redpanda broker while it's running, then you do a rolling restart which activates the new version on each broker as it comes back online?

Reply:
```suggestion
A rolling upgrade involves updating the version of Redpanda in the Redpanda  Helm chart, which triggers a rolling restart.
```